etc/org.opencastproject.organization-mh_default_org.cfg

```diff
@@ -251,6 +251,9 @@ prop.engageui.link_mobile_redirect.description=For more information have a look
 # disable the heartbeat
 # prop.player.matomo.heartbeat=30
 
+# Turn of notification that matomo is tracking the user.
```

Comment:
Setting this options to `true` will enable the notification and not turn it off, won't it? Also, stating the default value would be nice. Something like:

```
# Show notification that Matomo is tracking the user.
# Default: true
#prop.player.matomo.notification=true
```

Reply:
I updated the comment